app/src/main/java/org/jellyfin/mobile/player/source/JellyfinMediaSource2.kt

```diff
@@ -0,0 +1,93 @@
+package org.jellyfin.mobile.player.source
+
+import org.jellyfin.mobile.utils.Constants
+import org.jellyfin.sdk.model.api.MediaSourceInfo
+import org.jellyfin.sdk.model.api.MediaStream
+import org.jellyfin.sdk.model.api.MediaStreamType
+import org.jellyfin.sdk.model.api.PlayMethod
+import org.jellyfin.sdk.model.serializer.toUUIDOrNull
+import java.util.*
+import kotlin.collections.ArrayList
+
+class JellyfinMediaSource2(
```

Comment:
Not a fan of this name tbh

Reply:
Me neither. I'd prefer to get rid of the old one, but ExternalPlayer uses it xD

Reply:
Is it possible to migrate the externalplayer to this one?

Reply:
Need to check that, but could be possible.